src/components/AuthNav/AuthNav.tsx

```diff
@@ -0,0 +1,18 @@
+import { Link } from 'react-router-dom';
+
+import { ROUTES } from '../routes';
+
+import css from './AuthNav.module.css';
+
+const AuthNav: React.FC = () => {
+    return (
+        <div>
```

Comment:
I think you don't have to write this every time `React.FC`, typescript usually understand what you return. But you can discuss with team: do you want write type in all components or no. And write all components the same